examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -35,6 +39,18 @@ CHIP_ERROR PairingCommand::Run()
     GetExecContext()->commissioner->RegisterDeviceAddressUpdateDelegate(this);
     GetExecContext()->commissioner->RegisterPairingDelegate(this);
 
+#if CONFIG_PAIR_WITH_RANDOM_ID
+    // Generate a random remote id so we don't end up reusing the same node id
+    // for different nodes.
+    srandom(static_cast<unsigned>(System::Clock::GetMonotonicMicroseconds()));
+    NodeId randomId = static_cast<NodeId>(random());
```

Comment:
Suggest using `DRBG_get_bytes()` instead

Reply:
Oh, that is a good idea!  Will do that.

Reply:
Suggest a GenerateNodeId utility in CHIPCryptoPal or some util location, handles the 0 and 0xFFFF_FFFy_xxxx_xxxx special cases. Could be used as well for the examplary CA code. This could be done in a follow-up.

Reply:
Good idea; https://github.com/project-chip/connectedhomeip/issues/8461